client/editorQueryEvaluator.ts

```diff
@@ -210,7 +211,8 @@ export class EditorQueryEvaluator {
         resultsEditorTabIdentifier: Uri,
         editor: TextEditor
     ): void {
-        const contentType: contentType = workspace.getConfiguration().get('marklogic.sparqlContentType');
+        const queryType: ml.GraphsQueryType = getSparqlQueryForm(sparqlQuery);
```

Comment:
Because this command is hitting "/v1/graphs/sparql", and because the command is called "Eval SPARQL", we should consistently refer to this as "sparqlQuery" and not introduce "graphsQuery". The latter could be misinterpreted as referring to the "/v1/graphs" endpoint too which has a different set of inputs and output types.

So all occurrences of "GraphsQuery" should be renamed to "SparqlQuery" for consistency - e.g. `SparqlQueryType`.

Reply:
I aligned the type naming to be consistent with the existing graphs interface and rows format (RowsResponseFormat).  Should we change those as well?

Reply:
Hmm - "ResponseFormat" isn't great (the "format" of the response is something that conforms to the HTTP specification), but good point - let's stick with that for consistency, so yeah, `SparqlResponseFormat`. But I think we should definitely use "sparql" instead of "graphs" since this is specific to the "/v1/graphs/sparql" endpoint and not the "/v1/graphs" endpoint.